docs/DeveloperGuide.adoc

```diff
@@ -609,20 +625,24 @@ filter the car parks accordingly. The various predicates include
 Step 5. Next, `model` updates a filtered list of car parks by checking each car park
 against the specified predicate.
 
-Step 6. The following activity diagram summarizes what happens when a user executes a
+.How the filter operation works
+image::SequenceDiagram-filter.png[width="700", align="left"]
```

Comment:
Is there a mention to the sequence diagram?

Reply:
I felt that the steps actually describes the sequence diagram in this case, what do you think?

Reply:
I think we need to have reference to diagrams as per CS2101 style. @Lexzt any thoughts?